ebu_tt_live/bindings/__init__.py

```diff
@@ -1173,10 +1262,28 @@ def create_default_value(cls):
         )
         return instance
 
+    def _semantic_before_traversal(self, dataset, element_content=None, parent_binding=None):
+        self._semantic_register_id(dataset=dataset)
+
+    def _semantic_before_copy(self, dataset, element_content=None):
+        pass
+
+    def _validateBinding_vx(self):
+        origins = self.origin.split(" ")
+        extents = self.extent.split(" ")
+        if "%" not in self.origin:
```

Comment:
Is there an easy way to check that the value conforms to the defined XSD type `ebuttdt:pixelOriginType` instead of using this heuristic?

Reply:
if isinstance(self.origin, ebuttdt.percentageOriginType):
            raise InvalidRegionOriginType(self)